types/plotly.js/index.d.ts

```diff
@@ -481,7 +481,7 @@ export type ScatterData = PlotData;
 export interface PlotData {
 	type: 'bar' | 'box' | 'candlestick' | 'choropleth' | 'contour' | 'heatmap' | 'histogram' | 'mesh3d' |
 		'ohlc' | 'parcoords' | 'pointcloud' | 'scatter' | 'scatter3d' | 'scattergeo' | 'scattergl' |
-		'scatterternary' | 'surface';
+		'scatterternary' | 'surface' | 'pie' | 'scatterpolar';
```

Comment:
could you re-order this to maintain the alphabetical order of chart types - it's just easier to scan

Reply:
thanks for reviewing. I've fixed it.